src/js/components/Collapsible/__tests__/Collapsible-test.js

```diff
@@ -45,18 +45,31 @@ describe('Collapsible', () => {
     expect(container.firstChild).toMatchSnapshot();
     fireEvent.click(getByText('Toggle'));
     expect(open).toBe(true);
+    rerender(
+      <Grommet>
+        <Button
+          primary
+          onClick={() => {
+            open = !open;
+            useEffect();
+          }}
+          label="Toggle"
+        />
+        <Collapsible id="hidden-test" open={open}>
+          <Box>
+            <Text>Example</Text>
+          </Box>
+        </Collapsible>
+      </Grommet>,
+    );
     expect(container.firstChild).toMatchSnapshot();
     expect(useEffect).toBeCalledTimes(1);
-    fireEvent.click(getByText('Toggle'));
-    expect(open).toBe(false);
-    expect(container.firstChild).toMatchSnapshot();
-    expect(useEffect).toBeCalledTimes(2);
   });
 
-  test('vertical', () => {
+  test('rerender-vertical', () => {
```

Comment:
I think you should mention that its tests the `direction` prop.
So something like `renders direction vertical`.